src/main/java/second_calculator/strCalculator.java

```diff
@@ -0,0 +1,86 @@
+package second_calculator;
+import java.util.Scanner;
+
+public class strCalculator {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("입력 : ");
+
+        String str = sc.nextLine();
+
+        isBlank(str); //빈값여부
+
+        int result = calculator(str);
+
+        System.out.println("최종 값 : " + result);
+    }
+
+
+    public static int calculator(String str) {
+        String[] inputArray = str.split(" "); // 빈값제거
+        int result = Integer.parseInt(inputArray[0]);
+        for (int i = 1; i < inputArray.length; i=i+2) {
```

Comment:
```suggestion
        for (int i = 1; i < inputArray.length; i = i + 2) {
```
for 문 스페이스 컨벤션 지켜주세요!